wave_benchmark/src/trajectory_compare.cpp

```diff
@@ -0,0 +1,44 @@
+#include "wave/benchmark/trajectory_compare.hpp"
+
+namespace wave {
+
+void TrajectoryCompare::reset() {
+    this->ground_truth.clear();
+    this->measurements.clear();
+    this->error.clear();
+}
+
+void TrajectoryCompare::push_measurement(Pose pose, const TimeType time) {
+    measurements.insert(PoseMeasurement(time, this->measurement_key, pose));
```

Comment:
Okay. Only reason I used insert is because I didn't know what kind of argument emplace took.
```
template <typename T>
template <typename... Args>
std::pair<typename MeasurementContainer<T>::iterator, bool>
MeasurementContainer<T>::emplace(Args &&... args) {
// Support Boost.MultiIndex <= 1.54, which does not have emplace()
#if BOOST_VERSION < 105500
    return this->composite().insert(
      MeasurementType{std::forward<Args>(args)...});
#else
    return this->composite().emplace(std::forward<Args>(args)...);
#endif
}
```